client/ratings_reviews/components/ReviewsList.jsx

```diff
@@ -63,14 +63,22 @@ class ReviewsList extends React.Component {
   }
 
   render() {
-    return (
-      <div class='reviews-list'>
-        <div>{this.state.currentReviews.map(review => <ReviewTile review={review}/>)}</div>
-        <button class='review-show-button' onClick={this.showMoreReviews} hidden={!this.state.showMoreReviewsButton}>More Reviews</button>
-        <button class='review-show-button' onClick={this.showLessReviews} hidden={!this.state.showLessReviewsButton}>Less Reviews</button>
-      </div>
-    );
+    if (this.state.allReviews.length !== 0) {
+      return (
+        <div class='reviews-list'>
+          <div>{this.state.currentReviews.map(review => <ReviewTile review={review}/>)}</div>
+          <button class='review-show-button' onClick={this.showMoreReviews} hidden={!this.state.showMoreReviewsButton}>More Reviews</button>
+          <button class='review-show-button' onClick={this.showLessReviews} hidden={!this.state.showLessReviewsButton}>Less Reviews</button>
+        </div>
+      );
+    } else {
+      return (
+        <div>
+          Such emptiness! Be the first person to review this product!
```

Comment:
Would be nice to add a button to write a review from here!

Reply:
I think you may not need the 'else' statement because it's already returning something in the 'if' statement - just for future optimization :)

Reply:
Hi @jabrake, that's a fantastic point! I made a note of it, will change this when I refactor :)